Formula/pypy3.rb

```diff
@@ -3,6 +3,7 @@ class Pypy3 < Formula
   homepage "https://pypy.org/"
   url "https://bitbucket.org/pypy/pypy/downloads/pypy3.6-v7.3.0-src.tar.bz2"
   sha256 "48d12c15fbcbcf4a32882a883195e1f922997cde78e7a16d4342b9b521eefcfa"
+  head "https://foss.heptapod.net/pypy/pypy", :using => :hg, :branch => "py37"
```

Comment:
branch py3.6 or branch py3.7, or is this only informative?

Reply:
I chose 3.7 -- it's what will build when you run `brew install pypy3 --HEAD`. What do you think?

There's also a `--devel` we can use.

Reply:
Oh you're saying I typoed the dot too, yeah good catch. Will fix.

Reply:
I think 3.7 should be ok, does the name need to be the branch name? If so add a '.' : `py3.7`